heron/storm/src/java/shade.conf

```diff
@@ -0,0 +1,4 @@
+rule com.google.** org.apache.storm.__shaded__.@0
+rule com.twitter.** org.apache.storm.__shaded__.@0
```

Comment:
why would we shade com.twitter classes to the org.apache.storm namespace? I don't think we should do this.